app/src/main/java/nl/quintor/studybits/studybitswallet/AgentClient.java

```diff
@@ -42,7 +43,7 @@
     public AgentClient(String endpoint) {
         this.endpoint = endpoint;
     }
-    public ConnectionRequest login(String username, MessageEnvelopeCodec codec) {
+    public MessageEnvelope<ConnectionResponse> login(String username, MessageEnvelope<ConnectionRequest> envelope) {
```

Comment:
Shouldn't this need a password aswell? It won't be able to login to the dummy student 'Lisa' without a password.